prow/config.yaml

```diff
@@ -11806,6 +11806,145 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
 
+- interval: 2h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gci-gce-netd
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - "--timeout=340"
+      - "--repo=k8s.io/kubernetes=release-1.11"
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20180605-65a672b59-master
+      env:
+      - name: KUBE_CUSTOM_NETD_YAML
+        value: |2+
```

Comment:
This specifies that the default indentation is 2, but 4 spaces are used, so everything in this literal will be indented by 2 spaces. Is that intended? Usually the default indentation does not need to be specified. It can be inferred from the first line.

Reply:
we want to keep extra intended for the yaml.  thank you for raise the question. 
It will help us to skip a bug in pyyaml which will detect '---' from the yaml regardless of it belong to a quoted string. 
https://github.com/yaml/pyyaml/blob/master/lib/yaml/scanner.py#L1264

Reply:
Ok, it might be worth adding a comment to the yaml to explain this.

Reply:
Thanks, I have updated it.